int/hwsc-user-svc/test_insert.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO user_account (uuid, first_name, last_name, email, password, organization, create_timestamp)
```

Comment:
The schema is different now.

Reply:
I set defaults so I don't have to set it in user-service, unless i have to for some reason:
is_verified should always be false on insert
created_date is defaulted to current date on insert time

should i be explicitly setting it in user-svc instead?

Reply:
We should always manually set values in the service. Relying on a db makes it hard to debug later on.